src/registrar/tests/test_models.py

```diff
@@ -189,6 +198,24 @@ def test_transition_not_allowed_approved_investigating(self):
         with self.assertRaises(TransitionNotAllowed):
             application.in_review()
 
+    def test_transition_not_allowed_action_needed_investigating(self):
+        """Create an application with status action needed and call in_review
+        against transition rules"""
+
+        application = completed_application(status=DomainApplication.ACTION_NEEDED)
+
+        with self.assertRaises(TransitionNotAllowed):
+            application.in_review()
+
+    def test_transition_not_allowed_rejected_investigating(self):
```

Comment:
just a reminder, once the other one is merged anywhere it says investigating in this PR should change to in_review